nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/state/system-diagnostics/system-diagnostics.effects.ts

```diff
@@ -74,10 +87,12 @@ export class SystemDiagnosticsEffects {
                             }
                         })
                     ),
-                    catchError((error) =>
+                    catchError((errorResponse: HttpErrorResponse) =>
                         of(
-                            SystemDiagnosticsActions.systemDiagnosticsApiError({
-                                error: error.error
+                            ErrorActions.snackBarError({
+                                error: `Failed to load System Diagnostics. - [${
+                                    errorResponse.error || errorResponse.status
+                                }]`
                             })
```

Comment:
I think this needs to be using `systemDiagnosticsSnackbarError` otherwise the `status` stays in `loading`.